data/converters/converters.go

```diff
@@ -0,0 +1,195 @@
+package converters
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+func toConversionError(expected string, v interface{}) error {
+	return fmt.Errorf("expected %s input but got type %T for value \"%v\"", expected, v, v)
+}
+
+// Int64NOOP The input and output values are both int64.
```

Comment:
Like set, these will panic if the types are wrong, should call that out in the comment I think 

https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go/data?tab=doc#Field.Set

Reply:
```
// Int64NOOP is data.FieldConverter that performs no conversion.
// It is used when the input type is an int64 and the Field's type
// is also an int64.
// It will panic if the the input type does not match the Field type.
```

etc.